backend/models.py

```diff
@@ -51,14 +76,14 @@ class BookCreate(BookBase):
 class BookUpdate(SharedUpdate):
     title: Optional[str] = None
     author: Optional[str] = None
-    synopsis: Optional[str] = None
-    edition: Optional[str] = None
+    abstract: Optional[str] = None
+    publisher: Optional[str] = None
     catalog: Optional[str] = None
     category_type: Optional[str] = None
     category_age: Optional[str] = None
     category_topics: Optional[str] = None
     langage: Optional[str] = None
-    cover: Optional[str] = None
+    cover_url: Optional[str] = None
```

Comment:
any renaming of class's arguments should impact unit tests

Reply:
ok